packages/fxa-auth-server/lib/routes/subscriptions/paypal-notifications.ts

```diff
@@ -80,10 +86,39 @@ export class PayPalNotificationHandler extends PayPalHandler {
     }
   }
 
+  /**
+   * Remove the Paypal Billing Agreement
+   * If the Billing Agreement ID from the IPN request does not match the
+   * ID on the Stripe customer, then only update the database.
+   */
+  private async removeBillingAgreement(
+    customer: Stripe.Customer,
+    billingAgreement: PayPalBillingAgreements,
+    account: Account
+  ) {
+    if (
+      customer?.metadata[STRIPE_CUSTOMER_METADATA.PAYPAL_AGREEMENT] ===
```

Comment:
I think we have a helper function to get this.

Reply:
There is! Thank you.